policy/policy_handler/local.go

```diff
@@ -0,0 +1,49 @@
+package policy_handler
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/atomist-skills/go-skill"
+	"github.com/atomist-skills/go-skill/policy/data"
+	"olympos.io/encoding/edn"
+)
+
+const eventNameLocalEval = "evaluate_goals_locally"
+
+func WithLocal() Opt {
+	return func(h *EventHandler) {
+		h.subscriptionNames = append(h.subscriptionNames, eventNameLocalEval)
+		h.subscriptionDataProviders = append(h.subscriptionDataProviders, getLocalSubscriptionData)
```

Comment:
Maybe not something for now, but it would be interesting to look at the boundary between data from subscription and query and see whether we need such a clear separation - ultimately an "evaluator" doesn't really care where the data comes from...

Reply:
Agreed, although the policy repo needs to write the subscriptions and queries in separate places. Subscriptions spawn the execution, but queries are in-flight. I think it would be difficult to unify their logic in a way that is meaningful. But if you have an idea on how, I'm interested!

Reply:
Yeah, I don't know yet, but I'm imagining a world where the policy repo somehow defines subscriptions and queries orthogonally to the actual evaluation logic. Not something to worry about right now though.